src/common/createplay/CreatePlay.jsx

```diff
@@ -90,12 +93,14 @@ const CreatePlay = () => {
             }
           });
           setStoredData({ ...storedData });
-        })
-        .finally(() => {
-          setIsDataLoading(false);
+    }
+    catch(error){
```

Comment:
please explain the issue of previous code is having? and why `.finally` codes got moved in `.catch` ?